src/test/java/tassist/address/logic/commands/FindCommandTest.java

```diff
@@ -29,63 +35,106 @@ public class FindCommandTest {
 
     @Test
     public void equals() {
-        NameContainsKeywordsPredicate firstPredicate =
+        NameContainsKeywordsPredicate firstNamePredicate =
                 new NameContainsKeywordsPredicate(Collections.singletonList("first"));
-        NameContainsKeywordsPredicate secondPredicate =
+        NameContainsKeywordsPredicate secondNamePredicate =
                 new NameContainsKeywordsPredicate(Collections.singletonList("second"));
 
-        FindCommand findFirstCommand = new FindCommand(firstPredicate);
-        FindCommand findSecondCommand = new FindCommand(secondPredicate);
+        StudentId firstStudentId = new StudentId(VALID_STUDENTID_AMY);
+        StudentId secondStudentId = new StudentId(VALID_STUDENTID_BOB);
+        Predicate<Person> firstStudentIdPredicate = person -> person.getStudentId().equals(firstStudentId);
+        Predicate<Person> secondStudentIdPredicate = person -> person.getStudentId().equals(secondStudentId);
+
+        FindCommand findFirstNameCommand = new FindCommand(firstNamePredicate);
+        FindCommand findSecondNameCommand = new FindCommand(secondNamePredicate);
+        FindCommand findFirstStudenIdCommand = new FindCommand(firstStudentIdPredicate);
```

Comment:
Theres a t missing in student !
```suggestion
        FindCommand findFirstStudentIdCommand = new FindCommand(firstStudentIdPredicate);
```